### DISCUSSION TITLE
视频文件批量打包下载错误

### DISCUSSION THREAD

**Discussion by ghost** [spencerwooo/onedrive-vercel-index#621] [Q&A] [answered]
你好！

如题，在批量下载视频文件时会得到错误的文件（一个 2KB 大小的压缩包）。

**以下为复现方法：**

1. 进入[示例地址](https://xiaoxin1900drive.vercel.app/zh-CN/Test/)。
2. 勾选两个或者两个以上文件点击批量下载。
![image](https://user-images.githubusercontent.com/68503296/165326794-e1fb3a0c-3daf-4a4e-9214-202e972d14dc.png)
3. 下载完成后，会得到一个 zip 文件，大小一般为 2 KB，内容为视频缩略图，但格式为 mp4。
![image](https://user-images.githubusercontent.com/68503296/165327208-b918729e-3f13-49ee-bec9-23b223090560.png)

**附加信息：**
1. 批量下载时，有的时候也会直接卡在 0% 无法下载。
2. 以上内容在有无代理的情况下都经过测试。
3. [仓库地址](https://github.com/shincyan1900/onedrive-vercel-index)

请问这是因为什么原因导致的问题呢？👀

**Comment by myl7** [comment]
确实……我看看……

**Reply by myl7** [reply]
原因应该是 raw URL 最后的 + 应该 URL-encode 但没有做相应处理。

**Comment by myl7** [comment, marked as answer]
See #639

**Reply by ghost** [reply]
感谢！🌹
